EIPS/eip-2803.md

```diff
@@ -54,5 +54,11 @@ TBD.
 ## Implementation
 None yet.
 
+## Drawbacks and Alternative Proposals
+
+See https://github.com/ethereum/EIPs/issues/2863 for an alternative proposal.
+
+Particularly https://github.com/ethereum/EIPs/issues/2863#issuecomment-670551944 explains why this EIP is considered not a good idea.
```

Comment:
In general, external links shouldn't be included in an EIP and instead statements should be included inline.

Also, EIPs are not really the place for discussion on whether an EIP is a good idea or not, that goes better in the discussion-to link (see the header of the EIP).

Also, `## Drawbacks and Alternative Proposals` isn't one of the sections specified in the EIP template.  While the template doesn't have to be followed exclusively, it is generally desirable to ensure EIPs follow that layout.